packages/apps/ability-relay-link/test/e2e/smart-account/crossmint.spec.ts

```diff
@@ -0,0 +1,386 @@
+import {
+  setupVincentDevelopmentEnvironment,
+  getEnv,
+  type PkpInfo,
+  type VincentDevEnvironment,
+  type CrossmintSmartAccountInfo,
+} from '@lit-protocol/vincent-e2e-test-utils';
+import {
+  disconnectVincentAbilityClients,
+  getVincentAbilityClient,
+} from '@lit-protocol/vincent-app-sdk/abilityClient';
+import * as util from 'node:util';
+import { base } from 'viem/chains';
+import { entryPoint07Address } from 'viem/account-abstraction';
+
+import {
+  bundledVincentAbility as relayLinkAbility,
+  getRelayLinkQuote,
+  toVincentUserOp,
+  transactionsToCrossmintUserOp,
+  sendPermittedCrossmintUserOperation,
+} from '../../../src';
+
+jest.setTimeout(300000);
+
+// Check for required environment variables for Crossmint smart account testing
+const CROSSMINT_API_KEY = process.env.CROSSMINT_API_KEY;
+const SMART_ACCOUNT_CHAIN_ID = process.env.SMART_ACCOUNT_CHAIN_ID;
+const hasRequiredEnvVars = CROSSMINT_API_KEY && SMART_ACCOUNT_CHAIN_ID;
+
+(hasRequiredEnvVars ? describe : describe.skip)(
```

Comment:
We don't throw an error here because we're running the whole test suite? Seems like this should error if ran like `pnpm nx run ability-relay-link:test-e2e-smart-account:crossmint`? Same for other tests